src/app/core/header/header.component.ts

```diff
@@ -1,21 +1,25 @@
-import {Component, OnInit} from '@angular/core';
-import {ConfirmationService, MessageService} from "primeng/api";
-import {Router} from "@angular/router";
+import { Component, OnInit } from '@angular/core';
+import { ConfirmationService, MessageService } from 'primeng/api';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrl: './header.component.scss',
-  providers: [ConfirmationService, MessageService]
+  providers: [ConfirmationService, MessageService],
 })
-export class HeaderComponent implements OnInit{
-  position: string = 'center'
+export class HeaderComponent implements OnInit {
+  position: string = 'center';
 
-  constructor(private confirmationService: ConfirmationService, private messageService: MessageService, private router: Router) {
-  }
+  constructor(
+    private confirmationService: ConfirmationService,
+    private messageService: MessageService,
+    private router: Router
+  ) {}
 
   ngOnInit() {
-    localStorage.getItem('access_token')
+    //TODO: you do not use what you take here
+    localStorage.getItem('access_token');
```

Comment:
I'm not sure what was the idea, but it seems like you take value from localstorage and do not use it

Reply:
P.S. Ignore other changes, it's my VS Code autoformat